app/views/posts/index.html.erb

```diff
@@ -1,9 +1,10 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>User Posts</title>
-  </head>
-  <body>
-    <h1>Posts for a given User</h1>
-  </body>
-</html>
+<div>
+  <%= render partial: 'shared/user', locals: { user: @user } %>
+
+  <% @user.three_recent_posts.each do |post| %>
+    <%= render partial: 'shared/post', locals: { post: post } %>
+    <%= render partial: 'shared/comment', locals: { post: post } %>
```

Comment:
- [ ] @rmiyoyo Great work implementing the changes requested however as the previous reviewer noted you are displaying all comments instead of only displaying the five recent comments using `five_recent_comments` method, kindly consider implementing the suggested changes.